configs/generic_x86-64_hdd_config

```diff
@@ -38,3 +38,14 @@ MENDER_STORAGE_TOTAL_SIZE_MB=16000
 
 # Nothing to copy
 MENDER_COPY_BOOT_GAP="n"
+
+function platform_modify() {
+    #
+    # Make sure /lib64 exists since the Mender binary requires it.
+    # Some systems put everything under /lib (ie Yocto) and a simple
+    # symlink is enough to find everything Mender needs.
+    #
+    if [ ! -e work/rootfs/lib64 ]; then
+        run_and_log_cmd "ln -s /lib work/rootfs/lib64"
```

Comment:
It's early morning here, but will this not link `/lib` locally to the work directory. Is this right?

Reply:
Yes, but also to `/lib` on the device where the link is stored. :)

Reply:
I was almost sure I was wrong here. Just had to double check :p

Reply:
Clarification, the link will be `lib64 -> /lib` and the link is stored in `work/rootfs/` so on the device it should be `/lib64 -> /lib`. So I think this is correct.

But question is, why do we need this?

Reply:
It's because of [this bug](https://tracker.mender.io/browse/MEN-2799).